functions.py

```diff
@@ -42,42 +44,31 @@ def get_stats(text):
     avg_exc = tokens.count("!") / float(len(sentences))
 
     return {
-        "Mean words per sentence": awps,
-        "Sentence length SD": wpssd,
+        "Words p/s mean": awps,
+        "Words p/s SD": wpssd,
         "Lexical diversity": lex_diversity,
-        "Commas per sentence": avg_commas,
-        "Semicolons per sentence": avg_semi,
-        "Colons per sentence": avg_colons,
-        "Dashes per sentence": avg_dash,
-        "Dots per sentence": avg_dots,
-        "Question marks per sentence": avg_qm,
-        "Exclams per sentence": avg_exc,
+        "Commas p/s": avg_commas,
+        "Semicolons p/s": avg_semi,
+        "Colons p/s": avg_colons,
+        "Dashes p/s": avg_dash,
+        "Dots p/s": avg_dots,
+        "Question marks p/s": avg_qm,
+        "Exclamation marks p/s": avg_exc,
     }
 
 
-# TODO: that's def not the correct abstraction, just a placeholder to rewrite later
-def bootstrapped_ci(stats, lower=True, n=1000):
-    bts = []
+def bootstrapped_ci(text, q=0.5, n=1000, k=None):
+    random.seed(0)
+    sentences = nltk.sent_tokenize(text, language="russian")
+
+    stats = defaultdict(list)
+    if k is None:
+        k = len(sentences)
```

Comment:
I believe that's the only way to bootstrap. Then bootstrapped CI has a nice property of being a measure of uncertainty to an estimate.